docs/configuration/xml.rst

```diff
@@ -27,7 +27,7 @@ The basic steps to getting configuration set up with your application are:
 
 1. Set up your configuration in JSON or XML files that can be read by ``Microsoft.Extensions.Configuration``.
   * JSON Configuration uses ``Microsoft.Extensions.Configuration.Json``
-  * XML Configuration uses ``Microsoft.Extensions.Configuration.XML``
+  * XML Configuration uses ``Microsoft.Extensions.Configuration.Xml``
```

Comment:
Still need "configuration" in the sentence ("XML configuration" and "JSON configuration") to be lower case (not "JSON Configuration" - "Configuration" isn't a proper noun).